trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/extract-query-and-session-data.ts

```diff
@@ -46,9 +69,9 @@ export const extractQueryAndSessionData = (
   const isCheckboxChecked = getIsCheckboxChecked(selectedFeeRecordIds);
 
   return {
-    facilityIdQueryString,
-    filterError,
-    tableDataError,
+    premiumPaymentsFilters,
+    premiumPaymentsFilterError,
+    premiumPaymentsTableDataError,
```

Comment:
Similar to above, why do we want `premiumPayments` prefixes? We're already in the context of "reports", it seems like the original naming is simpler

In other words (and not for this PR), but I think e.g this:

```
controllers/utilisation-reports/utilisation-report-reconciliation-for-report/*
```

Could just be something like this:

```
controllers/utilisation-reports/reconciliation/*
```

I am not familiar with the functionality/URL etc, but I don't think we need to be as specific when in a child directory. What do you think?

Reply:
I agree that it could be good for this directory to be renamed.

Regarding the `premiumPayments` prefixes, this is due to us having multiple tabs on this page, each with their own error view models - please see this comment https://github.com/UK-Export-Finance/dtfs2/pull/3582/files#r1782538503